api/ctrl/src/buttons.rs

```diff
@@ -3,6 +3,71 @@ use core::fmt::Display;
 use core::ops::BitAnd;
 use sys::ffi::PDButtons;
 
+use crate::peripherals::{State, Buttons};
```

Comment:
it may look like a little circle, because for some reasons Buttons API was declared in `peripherals` module.